bucket/so.json

```
@@ -5,23 +5,23 @@
     "license": "MIT",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/samtay/so/releases/download/v0.4.7/so-v0.4.7-x86_64-pc-windows-msvc.zip",
-            "hash": "d66b044599b7f3baf85fd28f25d6a8fa2fef985994c2b1efed0948964d7bacde"
+            "url": "https://github.com/samtay/so/releases/download/v0.4.7/so-x86_64-pc-windows-msvc.zip",
+            "hash": "65e92c2af07e9cef3dc1eab47e63708d3420f504aca8058b2f04ccd64518cb11"
         },
         "32bit": {
-            "url": "https://github.com/samtay/so/releases/download/v0.4.7/so-v0.4.7-i686-pc-windows-msvc.zip",
-            "hash": "1b260a8257f1120db6bf3cebc4ddfa863f73f56afbcbaf68de1ffc1586c343eb"
+            "url": "https://github.com/samtay/so/releases/download/v0.4.7/so-i686-pc-windows-msvc.zip",
+            "hash": "5fb31d0b69c399f8e9c698123eb3d9036f6f28667b42a7a0500a087e5ed0ee14"
         }
     },
     "bin": "so.exe",
     "checkver": "github",
```

```suggestion
    "checkver": {
        "url": "https://github.com/samtay/so/releases",
        "regex": "v([\\d.]+)/so-x86_64-pc-windows-msvc\\.zip"
    },
```

`"checkver": "github"` only checks the **latest release** tag. (https://github.com/samtay/so/releases/latest)
Since *so* only has Windows builds in certain versions, we should match from all releases